### PR TITLE
ceph.spec: fix fedora pyopenssl package naming

### PR DESCRIPTION
**Pull request by noahdesu**

### CODE REVIEW

**Comment by tchaikov** [comment]
retest this please.

File: ceph.spec.in [hunk 1]
```diff
@@ -196,10 +196,10 @@ BuildRequires:	python%{_python_buildid}-requests
 BuildRequires:	python%{_python_buildid}-six
 BuildRequires:	python%{_python_buildid}-tox
 BuildRequires:	python%{_python_buildid}-virtualenv
-%if 0%{?fedora} || 0%{?rhel}
+%if 0%{?rhel}
 BuildRequires:  pyOpenSSL%{_python_buildid}
 %endif
-%if 0%{?suse_version}
+%if 0%{?fedora} || 0%{?suse_version}
```
Comment (tchaikov) [inline ceph.spec.in]: #26520 also takes RHEL8 into consideration. so i'm inclined to have @ktdreyer 's fix.

Comment (noahdesu) [reply ceph.spec.in]: sounds good to me!

Comment (noahdesu) [reply ceph.spec.in]: shall we close this?

Comment (tchaikov) [reply ceph.spec.in]: yeah, please!

**Pull request closed without merging**